src/client/results/mostfrequent/users/UserItemsList.js

```diff
@@ -14,6 +14,7 @@ class UserItemsList extends Component {
     $('.ui.dropdown.userwords').dropdown({
       action: 'nothing',
     });
+    $('.userItemsList').css('height', `${$(window).height() - ($(window).height() / 5)}px`);
```

Comment:
Extra point: Please can we call this code on a windowResized event or whatever it's called? Right now it keeps the height at 80% of the size of the window when the element first appears or the specific element re-renders